src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetPreviewTokenLogPage.tsx

```diff
@@ -81,7 +81,7 @@ export default function ReleaseApiDataSetPreviewTokenLogPage() {
                 <th>User</th>
                 <th>Date generated</th>
                 <th>Status</th>
-                <th>Expiry</th>
+                <th>Expires</th>
```

Comment:
Just the Robot test change to `public_api_preview_token.robot` that's still needed here I think?